test/integration/build_admission_test.go

```diff
@@ -127,8 +139,43 @@ func TestPolicyBasedRestrictionOfBuildConfigCreateAndInstantiateByStrategy(t *te
 	}
 }
 
-func buildStrategyTypes() []string {
-	return []string{"source", "docker", "custom", "jenkinspipeline"}
```

Comment:
I don't think this test is needed if you add the restricted/allowed check to the beginning of the previous test